wallet-new/src/Cardano/Wallet/API/Response.hs

```diff
@@ -43,6 +41,11 @@ import           Cardano.Wallet.API.Response.Sort.IxSet as SortBackend
 import           Cardano.Wallet.API.V1.Errors
                      (WalletError (JSONValidationFailed))
 
+import qualified Data.Aeson.Options as Serokell
+import qualified Data.Char as Char
+import qualified Formatting.Buildable
```

Comment:
Just a habit indeed, I usually do that when rebasing because it makes merge-conflicts easier to solve. Not sure we have any particular guidelines about this; I know it was suggested in the past but not clearly stated as the way to go.